### PR TITLE
PR number 1

### PR DESCRIPTION
**Pull request by chandan-g-bhagat**

### CODE REVIEW

**Review by Sailesharyal — COMMENTED** [review]
good

File: Basic/a.html [hunk 1]
```diff
@@ -4,10 +4,15 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document A</title>
+    <title>New File Test</title>
```
Comment (samir-maharjan) [inline Basic/a.html:7]: REmove the name test

File: Basic/a.html [hunk 2]
```diff
@@ -4,10 +4,15 @@
...
+    <h1>this is a change</h1>
+    <div>
+        <p>this is a change</p>
+    </div>
+    <div>The quick brown fox jumps over the lazy dog.</div>
 </body>
 <script></script>
-</html>
\ No newline at end of file
+<script></script>
```
Comment (samir-maharjan) [inline Basic/a.html:17]: Multiple blank script

File: Basic/b.html [hunk 3]
```diff
@@ -8,5 +8,7 @@
 </head>
 <body>
     
+
```
Comment (samir-maharjan) [inline Basic/b.html:11]: Remove multiple blank lines

**Review by samir-maharjan — CHANGES_REQUESTED** [review]

**Review by samir-maharjan — APPROVED** [review]

**Review by Sailesharyal — APPROVED** [review]

**Pull request merged**